### PR TITLE
Remove warning on Python3. Update Python 3 supported versions

### PR DESCRIPTION
**Pull request by kosmos342**
closes https://github.com/yola/demands/issues/39

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -3,15 +3,14 @@ sudo: false
...
-install:
-- pip install tox
+python:
+  - "2.7"
+  - "3.5"
+  - "3.6"
+  - "3.7"
+  - "3.8"
 script:
-- tox
```
Comment (RayeN) [inline .travis.yml]: Why did we decide to remove `tox`?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,11 @@
 # [Changelog](https://github.com/yola/demands/releases)
 
+## 4.2.0
```
Comment (RayeN) [inline CHANGELOG.md]: Probably 5.0.0 (since we aren't backwards compatible)

File: .travis.yml (hunk 1 above)
Comment (kosmos342) [reply .travis.yml]: Anyway, we used tox in travis to run only for one env for every travis's envs:
```
- TOXENV=py27
- TOXENV=py33
- TOXENV=py34
- TOXENV=pypy
```

Comment (kosmos342) [reply .travis.yml]: And there were errors in https://travis-ci.org/yola/demands/builds/617385518

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,11 @@
 # [Changelog](https://github.com/yola/demands/releases)
 
+## 5.0.0
+
+* Remove deprecation warning on Python 3.
```
Comment (RayeN) [inline CHANGELOG.md]: `Remove deprecation warning in Python 3.` ?

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,11 @@
...
+* Add Python 3.5, 3.6, 3.7, 3.8 as supported.
+* Remove Python 3.3, 3.4 as supported.
```
Comment (RayeN) [inline CHANGELOG.md]: `Stop supporting Python 3.3 and 3.4`?

**Pull request merged**